### PR TITLE
Sidebar styling, media queries fixing for smaller devices

### PR DESCRIPTION
**Pull request by roo-shy**
Update on a sidebar styling: 

1st level expand:

 screen shot 2016-12-11 at 4 32 29 pm

2d level: 
screen shot 2016-12-11 at 4 38 06 pm

### CODE REVIEW

File: assets/styles/docs.scss [hunk 1]
```diff
@@ -16,39 +16,78 @@ body.frey.docs {
...
     }
+    
+    // ul.collapse li a {
+    //   font-weight: normal;
+    //   font-size: 16px;
+    //   text-transform: uppercase;
+    //   color: $color-brightorange;
+    // }
+      
+      a.collapse.in:before {
```
Comment (kvz) [inline assets/styles/docs.scss]: indentation

File: docs/decorators/clustering/consul_acl.html [hunk 2]
```diff
@@ -10,70 +10,75 @@
 
 <h2>Options</h2>
 <table>
-<tr><th>name</th><th>description</th><th>required</th><th>choices</th></tr>
```
Comment (kvz) [inline docs/decorators/clustering/consul_acl.html]: Just to make sure: It's cool to play around here, just note that any changes here will be overwritten by the importer

**Comment by kvz** [comment]
Ready for merge?

**Comment by kvz** [comment]
I think it looks really good :)

**Comment by roo-shy** [comment]
I just commited a few more changes that I was working on yesterday, I added a few colors and started styling tables. Travis CI needs to pass and after it's good to go.

**Pull request merged**